soc/arm/arm/mps2/soc.c

```diff
@@ -37,7 +37,7 @@ FPGAIO_INIT(2);
 /* The base address that the application image will start at on the secondary
  * (non-TrustZone) Cortex-M33 mcu.
  */
```

Comment:
Ruh roh. Not in scope for this PR, but this really ought to come from the DT, it looks like.

Reply:
I agree, but this needs further rework beyond just the memory map and pulling values in from DT. I'll put together a more complete PR once I can allocate more time to it.

Reply:
@mbolivar-nordic This is already causing issues. Can you make a suggestion on the 'correct' was to get `CPU1_FLASH_ADDRESS` from `code: memory@38B000` in the `_remote` target?